ql/src/test/queries/clientnegative/bucket_mapjoin_mismatch1.q

```diff
@@ -1,46 +0,0 @@
-set hive.strict.checks.bucketing=false;
```

Comment:
there are a few tests which are being removed - will we miss them ?

Reply:
Removed tests are to test invalid hints which are not supported on Tez. It's been a long time since Tez is default engine. Nobody seems to miss them.